apps/api/src/report/report.util.service.ts

```diff
@@ -325,6 +325,45 @@ export class ReportUtilService {
         )`;
   }
 
+  getApplicantsWithStatus(
+    statuses: string[],
+    exclude: string[],
+    BCCNM_NEW_PROCESS: boolean,
+    to: string,
+    from: string,
+  ) {
+    const exclusion_query = ` AND NOT EXISTS (
+      SELECT *
+      FROM 
+        "ien_applicant_status_audit" AS T2
+      WHERE 
+        T2.applicant_id = status_audit.applicant_id
+        AND T2.status_id IN (
+          \'${exclude.join("','")}\'
+        )
+    )`;
+
+    let query = `SELECT 
+          count(distinct status_audit.applicant_id), status_audit.status_id
+        FROM 
+          "ien_applicant_status_audit" "status_audit"
+        LEFT JOIN 
+          ien_applicants applicant ON applicant.id = status_audit.applicant_id  
+        WHERE 
+          applicant.new_bccnm_process IS NOT NULL 
+          AND applicant.new_bccnm_process = ${BCCNM_NEW_PROCESS} 
+          AND "status_audit"."status_id" 
+          IN (
+            \'${statuses.join("','")}\'
```

Comment:
does this ened to be an array? just looks like we're passing a single value

Reply:
Yup. I'll update it. Originally, I intended this function to be able to handle multiple statues, (eg do all full licences at the same time)